packages/tezt-bam/tezt-bam.0.2/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "A plugin of [bam] for Tezt"
+description: "Provides a way to register PBT tests with Tezt"
+maintainer: ["François Thiré"]
+authors: ["François Thiré"]
+license: "MIT"
+tags: ["test" "tezt" "pbt" "shrinking" "internal" "bam"]
+homepage: "https://github.com/francoisthire/bam"
+doc: "https://francoisthire.github.io/bam/"
+bug-reports: "https://github.com/francoisthire/bam/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.7" & >= "3.7"}
+  "tezt" {>= "4.0"}
+  "bam"
+  "bam-ppx"
+  "mtime" {>= "2.0"}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
]
conflicts: [
  "ocaml-option-bytecode-only"
]
```

We can publish the package with this (if the CI passes after it's applied). You should still use the template thing upstream to avoid having to add this in future releases.

Reply:
hmm I might have gotten the specific option wrong. It's a 32 bit issue.